kolibri/plugins/learn/assets/src/vue/side-nav/index.vue

```diff
@@ -93,6 +126,10 @@
   svg
     fill: $core-action-normal
     transition: fill $core-time ease-out
+    @media screen and (orientation: portrait)
```

Comment:
we can probably choose a good `min-width` breakpoint based on the card width and gutter variables, similar to how we do the other breakpoints. See e.g. https://github.com/learningequality/kolibri/blob/master/kolibri/plugins/learn/assets/src/vue/learn.styl#L22